src/dealDashboard/dealDiscussion/discussionThread/discussionThread.html

```diff
@@ -107,6 +107,7 @@ <h3 class="subtext">
           type="primary"
           full-width
           disabled.bind="isLoading.commenting || !comment.length"
+          is-loading.bind="true"
```

Comment:
Why is it here?
Maybe something like:
`is-loading.bind="isLoading.commenting || isLoading.replying"`?